### ISSUE TITLE
Feature Request: Option to set up "Up" and "Down" arrows shortcuts

### ISSUE THREAD

**Issue by r-menezes** [willsALMANJ/Zutilo#157]
If not too complicated to implement, this would allow for quicker navigation by using vim's J and K keys for down and up respectively.

PS: Thank you a lot for this crucial extension!

**Comment by wshanks** [comment]
Yes, these would be good to have. Back when Zotero was a Firefox extension, I had written a plugin for another Firefox extension (Pentadactyl) that provided keyboard shortcut functionality to implement UI navigation commands like this. The functions it used for this might still work in Zotero now if they were ported to Zutilo: https://github.com/wshanks/Zoterodactyl

**Comment by Mingzefei** [comment]
Is this feature still in the development plan? 
I really hope to be able to use vim keybindings with Zotero!
